part_2_dynamic.md

```diff
@@ -4,66 +4,95 @@ title: Part 2
 permalink: /part_2/
 ---
 
-# Part 2 - Creating the Rails app
+# Part 2 - Introducing Rails
 
-* Open the terminal and create a new Rails app
+## Creating your (first) Rails app
+
+Open the terminal to create a new Rails app
 
 {% highlight bash %}
-rails new railsgirls-app
-cd railsgirls-app
+rails new railsgirls-blog
+cd railsgirls-blog
 {% endhighlight %}
 
-* Have a look how Rails generated a folder structure and a lot of files for you. There are two ways of doing that, in the terminal or with Finder/Explorer. Ask your coach about the details. Once you have seen them, move your ```about.html``` to the public folder.
+Find the project directory in the Finder/Explorer. Have a look how Rails generated a folder structure and a lot of files for you. Ask your coach about the details.
+
+
+## Starting the Rails server
+
+A Rails app can deliver dynamically generated content with the help of the Ruby programming language. This is why we can't just open a file anymore in the browser. We need a Rails server that will allow for all the magic to happen.
```

Comment:
This is unclear. They even probably dont know what a server is. I would rephrase saying something like: «In order to view the files, we need a server. It will get what info needs to be fetched from the tremplate, then inject the info and create the final file as html». Probably can be rephrased as well.